test/sample-test/configs/component_build.config.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+test_name: component_build
+notebook_params:
+  EXPERIMENT_NAME: component_build-test
```

Comment:
recommend using the lower case experiment_name since it is replaced by the test infra no matter what value is here. @numerology

Reply:
Fixed. Thanks for the info.